plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/ext/provider/WsAgentVolumeProvider.java

```diff
@@ -68,6 +71,10 @@ public String get() {
     }
 
     private String getTargetOptions(final String path) {
-        return path + CONTAINER_TARGET + ":" + volumeOptions;
+        if (!Strings.isNullOrEmpty(agentVolumeOptions)) {
```

Comment:
Please create constructor and put this code there as you did in LocalCheInfrastructureProvisioner